concepts/review.md

```diff
@@ -1,5 +1,103 @@
-# Work in Progress
+# Getting Analysis Review
+
+The current process for getting analysis review has not been optimized.
+If you run into problems, feel free to ask #datapipeline
+
+## Table of Contents
+
+<!-- toc -->
+
+# Finding a Reviewer
+
+There are two major types of review for most analyses:
+**Data review** and **Methodology review**.
+
+A **data reviewer** has **expert understanding of the dataset** your analysis is based upon.
+During data review, your reviewer will try to identify
+any issues with your analysis that come from misapplying the data.
+For example, sampling issues or data collection oddities.
+
+**Methodology review** primarily covers **statistical concepts**.
+For example, if you're training regressions, forecasting data, or doing complex
+statistical tests, you should probably get Methodology review.
```

Comment:
Perhaps we can make a slightly more explicit (possibly almost exhaustive) list of _types_ of statistical analyses that are common in Mozilla so that people are not wondering whether their particular analysis requires methodological review or not. 

I.e. I'm doing a clustering analysis on some categorical feature space. Do i need to get this reviewed for methodology?